src/app/examples/list-example/base-list.component.ts

```diff
@@ -2,13 +2,13 @@ export class BaseListComponent {
   myItems = [
     {
       title: 'Vestas Wind Systems has a very long name',
-      subTitle: '2000 stk',
+      subTitle: '2000 pcs',
```

Comment:
I would not touch this, as it has nothing to do with the fix. Could you revert it?